CI/JenkinsfileTestCleanInstance

```diff
@@ -215,17 +215,45 @@ pipeline {
                         script {
                             docker.withRegistry("", 'polycube-repo') {
                                 sh """
-                                    docker stop polycubed || true
+                                    container="polycubed"
+                                    docker stop \$container || true
                                     docker system prune --all --force
                                     # Launch the polycubed daemon and extract the polycubectl
                                     # /proc is mounted under /host/proc to have a complete visibility of the namespaces created outside of the container
-                                    docker run -d --name polycubed --rm --privileged --pid=host --cap-add ALL --network host -v /proc:/host/proc -v /lib/modules:/lib/modules:ro -v /var/run/netns/:/var/run/netns:rw -v /usr/src:/usr/src:ro -v /etc/localtime:/etc/localtime:ro polycubebot/${image_name}-default:${image_tag}
-                                    mkdir bin
-                                    docker cp polycubed:/usr/local/bin/polycubectl ./bin/
+                                    docker run -d --name \$container --rm --privileged --pid=host --cap-add ALL --network host -v /proc:/host/proc -v /lib/modules:/lib/modules:ro -v /var/run/netns/:/var/run/netns:rw -v /usr/src:/usr/src:ro -v /etc/localtime:/etc/localtime:ro polycubebot/${image_name}-default:${image_tag}
+                                    
+                                    # Copying binaries

```

Comment:
Sorry for the silly question: is there any way to define this code ones, such as with a function? It looks we have a lot of duplicated code here.